app.py

```diff
@@ -117,7 +125,6 @@ def change_password():
     if confirm_new_password != new_password:
```

Comment:
where is the logic of checking whether the old password is correct or not?
Also, add logic to ensure that new password and confirm password fields are not left blank.